module/zfs/spa.c

```diff
@@ -4114,12 +4114,6 @@ spa_import(char *pool, nvlist_t *config, nvlist_t *props, uint64_t flags)
 
 	VERIFY(nvlist_lookup_nvlist(config, ZPOOL_CONFIG_VDEV_TREE,
 	    &nvroot) == 0);
-	if (error == 0)
-		error = spa_validate_aux(spa, nvroot, -1ULL,
-		    VDEV_ALLOC_SPARE);
-	if (error == 0)
-		error = spa_validate_aux(spa, nvroot, -1ULL,
-		    VDEV_ALLOC_L2CACHE);
```

Comment:
Dropping these checks will result in the shared aux device tests being disabled during import.  That shouldn't be a problem since these checks only make sense during `zpool create` and `zpool add`.  At import time we shouldn't be re-initializing any devices and instead only verifying that the required devices all have valid labels,